featuremanagement/_featurefilters.py

```diff
@@ -12,36 +23,36 @@ class FeatureFilter(ABC):
     """
 
     @abstractmethod
-    def evaluate(self, context, **kwargs):
+    def evaluate(self, context: Mapping, **kwargs: Unpack[RequestParams]) -> bool:
```

Comment:
I don't know whether my current understanding is correct. If I say something stupid please forgive me. 

IMO, `**kwargs` itself is a `dict` already. For `foo(**kwargs)`, you can do something like: `for key, value in kwargs.items(): # do something#`

But now, you have `foo(**kwargs: Unpack[RequestParams])`. The way you call `foo` will be like: `foo(param1={"key": "A", "value": 1}, param2={"key": "B", "value": 2})`.

It will be `foo(param1={"key": "A", "value": 1}` vs  `foo(A=1)`

I think `RequestParams` class is over-engineered.

Reply:
I agree, I was looking to figure out what the correct value of it should be and the `RequestParams` solution is one of them. the other being a `Dict[str, Any]`. I'm good with either one, it seems like you lean towards the `Dict[str, Any]` option.